docs/source/community/governance.rst

```diff
@@ -1,128 +1,292 @@
+file in the GitHub repository. For other modules that correspond
+to repositories, membership is recorded on GitHub as access
+level to the repo (i.e. “write” permission). Module maintainers
+are given privileges to administrate the repository (except for
+`pytorch/pytorch <https://github.com/pytorch/pytorch>`__ where
+they are responsible for a folder).
+
+Core Maintainers
+----------------
+The core maintainers are expected to have a deep understanding
+of the PyTorch code base and design philosophies. Their responsibilities
+include:
+
+* Articulating a cohesive long-term vision for the project
+* Negotiating and resolving contentious issues in ways
+  acceptable to all parties involved
+* Receiving broad requests for changes from stakeholders of
+  PyTorch and evaluating / accepting them (small module-level
+  requests are handled by module maintainers)
+
+The core maintainers as a group have the power to veto any
+decision made at a Module maintainer level. The core
+maintainers have power to resolve disputes as they see fit.
+The core maintainers should publicly articulate their
+decision-making, and give a clear reasoning for their
+decisions, vetoes and dispute resolution.
+
+The core maintainers are admins of the PyTorch GitHub Org
+and are listed in `Persons of Interest <https://pytorch.org/docs/stable/community/persons_of_interest.html>`__.
+
+Lead Core Maintainer (BDFL)
+---------------------------
+
+There may be decisions in which the core maintainers cannot
+come to a consensus. To make such difficult decisions, the
+core maintainers have an assigned and publicly declared Lead
+Core Maintainer amongst them, also commonly known in open-source
+governance models as a BDFL.
+
+The Lead Core Maintainer should publicly articulate their
+decision-making, and give a clear reasoning for their
+decisions. The Lead Core Maintainer is also responsible for
+confirming or removing core maintainers.
+
+Nominating, Confirming and Removing Maintainers
+-----------------------------------------------
+
+The Principles
+++++++++++++++
+
+* Membership in module maintainer groups is given to **individuals**
+  on **merit basis** after they demonstrated strong expertise of the
+  component through contributions, reviews and discussions and are
+  aligned with how the component fits in overall PyTorch direction.
+* For membership in the maintainer group the individual has to
+  demonstrate strong and continued alignment with the overall
+  PyTorch principles.
+* No term limits for module maintainers or core maintainers
+* Light criteria of moving module maintenance to ‘emeritus’
+  status if they don’t actively participate over long periods
+  of time. Each module maintainer group may define the inactive
+  period that’s appropriate for that module.
+* The membership is for an individual, not a company.
+
+The Process for Nomination
+++++++++++++++++++++++++++
+
+* We have a `nomination form <TBA>`__, where anyone in the community can
```

Comment:
Should we change this to `We will have a nomination form, where...` and no link for now? @b0noI we are still blocked on the form, correct? Thanks.

Reply:
@svekars can you update to the above suggestion please and then we can land this? Thanks!